diff --git a/iMage/jmjrst.main/src/main/java/org/jis/generator/LayoutGalerie.java b/iMage/jmjrst.main/src/main/java/org/jis/generator/LayoutGalerie.java
index 3e9a4e2..2671145 100644
--- a/iMage/jmjrst.main/src/main/java/org/jis/generator/LayoutGalerie.java
+++ b/iMage/jmjrst.main/src/main/java/org/jis/generator/LayoutGalerie.java
@@ -549,7 +549,7 @@ public class LayoutGalerie {
   public void copyFile(File file, File ziel) throws FileNotFoundException, IOException
   {
     in = new BufferedInputStream(new FileInputStream(file));
-    out = new BufferedOutputStream(new FileOutputStream(ziel, true));
+    out = new BufferedOutputStream(new FileOutputStream(ziel, false));
     int bytes = 0;
     while ((bytes = in.read()) != -1)
     {